cuav/image/scanner.c

```diff
@@ -856,7 +867,18 @@ static void mark_regions(struct bgr_image *img, const struct regions *r)
 	}
 }
 
-
+/*
+  map a value based on the threshold
+ */
+uint8_t map_value(float v, const float threshold) {
```

Comment:
should be static and const?

Reply:
gcc gives compiler errors if I try that -
``error: storage class specified for parameter``